youtube_dl/extractor/zdf3sat.py

```diff
@@ -39,7 +39,7 @@ def _extract_player(self, webpage, video_id, fatal=True):
 
 
 class ZDFIE(ZDFBaseIE):
-    _VALID_URL = r'https?://www\.zdf\.de/(?:[^/]+/)*(?P<id>[^/?]+)\.html'
+    _VALID_URL = r'https?://www\.(?:zdf|3sat)\.de/(?:[^/]+/)*(?P<id>[^/?]+)\.html'
```

Comment:
Not all URLs are interchangeable so it's most likely should be separate extractor.

Reply:
Understood. Made new 3sat extractor based on zdf extractor.

Reply:
> Not all URLs are interchangeable so it's most likely should be separate extractor.

Interesting, would you mind giving an example?